docker-compose.yml

```diff
@@ -7,42 +7,44 @@ services:
     build:
       context: .
       dockerfile: Dockerfile-dev
-    command: script/docker-app.sh
+    command: docker/dev/run.sh
     environment:
-      DB_HOST: ${DB_HOST}
-      DB_USER: ${DB_USER}
-      DB_PASSWORD: ${DB_PASSWORD}
-      DB_NAME: ${DB_NAME}
+      DB_HOST: mysql
+      DB_NAME: portal
+      DB_USER: master
+      DB_PASSWORD: master
       SOLR_HOST: solr
       SOLR_PORT: 8983
-    networks:
-      - appnet
     ports:
-      - "3000:3000"
+      - 3000:3000
     volumes:
       - .:/rigse
-      - "bundle:/bundle"
+      - bundle:/bundle
   solr:
-    image: solr:5.3
-    networks:
-      - appnet
+    image: concordconsortium/docker-solr-portal
     ports:
-      - "8983:8983"
+      - 8983:8983
     volumes:
       - .:/rigse
       - data:/opt/solr/server/solr/mycores
     depends_on:
       - solrdata
-    command: /rigse/docker/start-solr-within-solr-container.sh
+    command: /bin/bash ./start-solr.sh
   solrdata:
-    image: solr:5.3
+    image: concordconsortium/docker-solr-portal
     volumes:
       - data:/opt/solr/server/solr/mycores
     user:
       root
-    command: chown solr:solr /opt/solr/server/solr/mycores
-networks:
-  appnet:
+    command: /bin/bash ./start-volume.sh
+  mysql:
```

Comment:
It looks good, thanks!
I think that Docker docs mentioned `.env` file as a way to provide default values. But that means that `.env` would need to be checked in and would make it more annoying if you wanted to use it for your own env variables. So separate compose file makes more sense. 👍